src/persistence.jl

```diff
@@ -1 +1,7 @@
 # Here put file load/file write functions
+function loadReference(path)
+    f = FASTA.Reader(open(path,"r"))
+    for r in f
```

Comment:
as fasta can have multiple sequences we want to return to the user all of them, so output should be a vector of these sequences

also you are not closing anywhere open file f